hs_core/hydroshare/utils.py

```diff
@@ -397,7 +397,10 @@ def copy_resource_files_and_AVUs(src_res_id, dest_res_id):
 
     for n, f in enumerate(files):
         folder, base = os.path.split(f.short_path)  # strips object information.
-        new_resource_file = ResourceFile.create(tgt_res, base, folder=folder)
+        if f.reference_file_path is not None:
+            new_resource_file = ResourceFile.create(tgt_res, base, folder=folder, is_file_reference=True)
+        else:
+            new_resource_file = ResourceFile.create(tgt_res, base, folder=folder)
```

Comment:
The line 400-403 could be simplified into one line like so: ```new_resource_file = ResourceFile.create(tgt_res, base, folder=folder, is_file_reference=True if f.reference_file_path else False``` For your reference.

Reply:
@ismagilani Did you see my comment above? I think the logic is clearer if you make the simplification as I suggested above, but your code should work as well. I am approving the PR and defer to you whether you want to make this code simplification or not.